src/components/LoadButton/LoadButton.scss

```diff
@@ -0,0 +1,21 @@
+.actions {
```

Comment:
filename must be the same as the block name

Reply:
good point